client/src/components/layout/Page.jsx

```diff
@@ -1,14 +1,21 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
+import type { Node } from 'react';
 import '../../css/Page.css';
 
+type PageProp = {
+  title: any,
+  children: any,
+  menuItems?: any,
```

Comment:
Are there more specific types we can use here?

Reply:
Yes, was going to deep dive to avoid generic typing (`any`, `Object`, `Function`) in another PR. wanted to limit the size of this PR as it was already getting big